### PR TITLE
Fix: use CodeVerification component for login 2fa form

### PR DESCRIPTION
**Pull request by seanyellowd**
https://openware-inc.atlassian.net/browse/OPSM-2307

### CODE REVIEW

File: web/src/components/TwoFactorAuth/index.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export const TwoFactorAuthComponent: React.FC<TwoFactorAuthProps> = ({
                                 'cr-email-form__group--focused': codeFocused,
                             })}>
                             <CustomInput
-                                type="number"
+                                type="text"
```
Comment (josadcha) [inline web/src/components/TwoFactorAuth/index.tsx:72]: we have a new 2fa form input on beneficiary submit modal. Could you please find that component and reuse it here. It looks like the 2fa input in yellow

**Comment by akhlopiachyi** [comment]
https://github.com/openware/baseapp/tree/master/web/src/components/CodeVerification

File: web/src/components/TwoFactorAuth/index.tsx (hunk 1 above)
Comment (seanyellowd) [reply web/src/components/TwoFactorAuth/index.tsx]: pushed the code.

File: web/src/containers/ProfileApiKeys/index.tsx [hunk 2]
```diff
@@ -352,7 +352,7 @@ class ProfileApiKeysComponent extends React.Component<Props, ProfileApiKeysState
                 </div>
                 <div className={emailGroupClass}>
                     <CustomInput
-                        type="number"
+                        type="text"
```
Comment (josadcha) [inline web/src/containers/ProfileApiKeys/index.tsx:355]: the idea was to reuse here the CodeVerification component. Isn't it possible

**Review by josadcha — APPROVED** [review]

**Review by akhlopiachyi — APPROVED** [review]

**Pull request merged**